### PR TITLE
IDEX-3084: Added ability to select recipe during creation of workspace

### PR DESCRIPTION
**Pull request by dimasnurenko**
@vparfonov

### CODE REVIEW

**Comment by skabashnyuk** [comment]
You need to update this PR

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/createworkspace/CreateWorkspaceViewImpl.java [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+class CreateWorkspaceViewImpl extends Window implements CreateWorkSpaceView, TagEntry.ActionDelegate {
+
+    interface CreateWorkspaceViewImplUiBinder extends UiBinder<Widget, CreateWorkspaceViewImpl> {
+    }
+
+    private static final CreateWorkspaceViewImplUiBinder UI_BINDER = GWT.create(CreateWorkspaceViewImplUiBinder.class);
+
+    private static final int BORDER_WIDTH = 1;
+
+    static final String RECIPE_URL =
```
Comment (skabashnyuk) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/createworkspace/CreateWorkspaceViewImpl.java]: what is it?

File: ide/che-core-ide-app/src/main/resources/org/eclipse/che/ide/workspace/recipe.svg [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (skabashnyuk) [inline ide/che-core-ide-app/src/main/resources/org/eclipse/che/ide/workspace/recipe.svg]: Who are the author of this file?

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/createworkspace/CreateWorkspaceViewImpl.java (hunk 1 above)
Comment (dimasnurenko) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/createworkspace/CreateWorkspaceViewImpl.java]: What do you mean?

File: ide/che-core-ide-app/src/main/resources/org/eclipse/che/ide/workspace/recipe.svg (hunk 2 above)
Comment (dimasnurenko) [inline ide/che-core-ide-app/src/main/resources/org/eclipse/che/ide/workspace/recipe.svg]: All icons will be reworked, and we decided let it be as it is for now.

**Pull request closed without merging**